doc/alps_submitter.md

```diff
@@ -0,0 +1,144 @@
+# ALPS Submitter
+
+## Precondition
+1. Custom estimator is not supported.
+1. Train with evaluate is not supported.
+1. Inputs needs to set the type of the dense/sparse, shape, etc.
+
+## Data Pipeline        
+Standard Select -> Train Input Table -> Parser -> Input Fn and TrainSpec
+
+Parser is implemented in `COLUMN`, using expr like `DENSE`, `SPARSE`, `IDENTITY_SPARSE`.
+
+Data Example
+1. numeric feature：float
+1. dense feature: "0.1, 2.9, 20, 13.2"
+1. sparse feature:  "city: hangzhou, sex: male, interest: book"
+1. identity sparse feature："1, 100, 2000, 31"
+
+```sql
+select 
+  c1, c2, c3, c4, c5 as class
+from kaggle_credit_fraud_training_data
+TRAIN DNNClassifier
+WITH
+  batch_size = 512
+  train_spec.max_step = 100
+  ...
+COLUMN
+  DENSE(c1, shape=[5], dtype=float)
```

Comment:
Should `DENSE(c1, shape=[5], dtype=float)` include `separator=","`?

If not, does it mean all dense features are encoded in a string and separated by commas?